.github/workflows/runOscalcliValidation.yml

```
@@ -0,0 +1,34 @@
+name: Run OSCAL CLI Validation
+
+on:
+  pull_request:
+    branches: [ main, master ]  # Adjust these branch names as needed
+  workflow_dispatch:  # Keeps the manual trigger option
+ 
+jobs:
+  run-oscal-cli-validation:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
```

According to [this guide](https://whichjdk.com/#:~:text=The%20former%20AdoptOpenJDK%20project%20has,tested%20under%20a%20permissive%20license.), `temurin` is one of the better supported.

```suggestion
        distribution: 'temurin'
        cache: 'maven' # cache maven dependencies
```